tests/test_rigid_physics.py

```diff
@@ -466,3 +466,41 @@ def test_equality_joint(gs_sim, mj_sim):
     # check if the two joints are equal
     gs_qpos = gs_sim.rigid_solver.qpos.to_numpy()[:, 0]
     np.testing.assert_allclose(gs_qpos[0], gs_qpos[1], atol=1e-9)
+
+
+@pytest.mark.parametrize("backend", [gs.cpu], indirect=True)
+def test_urdf_mimic_panda(show_viewer):
+    # create and build the scene
+    scene = gs.Scene(
```

Comment:
Just use `gs_sim`. No need to create the whole scene yourself.

Reply:
Ok for now you have to create the scene yourself because `gs_sim` depends on `mj_sim`... It would be nice to make the dependency optional to avoid having the create the scene manually.

Still remove options that are not important for the unit test.

Reply:
deleted unused option